src/screens/Library/AllGroups/AllGroups.scss

```diff
@@ -41,19 +53,15 @@
     overflow: hidden;
     white-space: nowrap;
     max-width: 250px;
-
```

Comment:
Are this line removals intended?

Reply:
No, but I can't undo it due to the eslint configuration